src/navigators/Main.js

```diff
@@ -33,6 +34,8 @@ import BrowseSettings from '../screens/browse/SettingsBrowseScreen';
 const Stack = createStackNavigator();
 
 const MainNavigator = () => {
+  useKeepAwake();
```

Comment:
It would be better to call the hook in ReaderScreen instead of MainNavigator.

Reply:
Yes Its will be a better choice